.github/workflows/lint.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - uses: ros-tooling/setup-ros@v0.2
     - uses: ros-tooling/action-ros-lint@v0.1
       with:
-        distribution: foxy
+        distribution: rolling
```

Comment:
This is a little dangerous, we'd subject the repo to ever-changing format requirements?